fe/fe-core/src/main/java/org/apache/doris/catalog/MetaIdGenerator.java

```diff
@@ -41,16 +43,27 @@ public void setEditLog(EditLog editLog) {
 
     // performance is more quickly
     public synchronized long getNextId() {
-        if (nextId < batchEndId) {
-            return nextId++;
-        } else {
+        if (nextId >= batchEndId) {
             batchEndId = batchEndId + BATCH_ID_INTERVAL;
             if (editLog != null) {
                 // add this check just for unit test
                 editLog.logSaveNextId(batchEndId);
             }
-            return nextId++;
         }
+        return nextId++;
+    }
+
+    public synchronized IdGeneratorBuffer getIdGeneratorBuffer(long bufferSize) {
+        Preconditions.checkState(bufferSize > 0);
+        IdGeneratorBuffer idGeneratorBuffer = new IdGeneratorBuffer(nextId, nextId + bufferSize - 1);
```

Comment:
I'm a little confused, It looks like we're getting ids in batches, Is the number of Editlog calls the same as before？The reason for the improvement is that getNextId no longer requires synchronized or reduce the number of Editlog calls？

Reply:
the time cost for meta management should be as small as possible, because the editlog is shared by the whole cluster, when there are many dbs, tables, tablets, it is easy to see poor performance for meta-management